src/supply/supply.controller.ts

```diff
@@ -31,6 +32,23 @@ export class SupplyController {
     }
   }
 
+  @Get('top')
+  async top(@Query('limit') limit: number = 10, @Query('skip') skip: number) {
+    try {
+      if (limit && typeof limit === 'string') {
```

Comment:
é só usar o pipe ParseIntPipe

see: https://docs.nestjs.com/pipes#built-in-pipes

Reply:
Boa, ajutado!